pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetActivityInstanceRoute.java

```diff
@@ -127,6 +125,27 @@ public Object handle(Request request, Response response) {
         return result;
     }
 
+    private Optional<ActivityInstance> getActivityInstance(
+            String userGuid,
+            String studyGuid,
+            String instanceGuid,
+            String operatorGuid,
+            Handle handle,
+            ActivityInstanceDto instanceDto,
+            ContentStyle style,
+            String isoLangCode) {
+
+        Optional<ActivityInstance> inst = actInstService.buildInstanceFromActivityDefStore(
+                handle, isoLangCode, style, studyGuid, userGuid, operatorGuid, instanceDto
+        );
+        if (inst.isEmpty()) {
+            inst = actInstService.getTranslatedActivity(
+                    handle, userGuid, operatorGuid, instanceDto.getActivityType(), instanceGuid, isoLangCode, style
+            );
+        }
```

Comment:
I don't think we should fallback to the old approach here. Once we switch to new approach, we should deprecate this. I look forward to deleting code for `getTranslatedActivity()` and related methods!